packages/design-system-docs/src/scripts/components/PageHeader.jsx

```diff
@@ -36,26 +54,27 @@ class PageHeader extends React.PureComponent {
   render() {
     return (
       <header className="ds-u-padding--3 ds-u-sm-padding--6 ds-u-display--block ds-u-fill--gray-lightest">
-        <h1
-          className="ds-display"
-          // eslint-disable-next-line react/no-danger
-          dangerouslySetInnerHTML={{ __html: this.props.header }}
-          id={this.props.reference}
-        />
-        <div className="ds-u-clearfix" />
-        <div className="ds-u-font-size--small">
+        <div className="ds-u-display--flex ds-u-align-items--center">
+          <h1
+            className="ds-display ds-u-display--inline-block"
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={{ __html: this.props.header }}
+            id={this.props.reference}
+          />
           {this.statusBadge()}
-
+        </div>
+        <div className="ds-u-font-size--small">
           {this.guidanceLink()}
-
           {this.uswdsLink()}
+          {this.cmsdsLink()}
```

Comment:
Could we put the CMSDS link first? We probably need to redo the logic here as well. Instead of trying to chain the two links together with two `span` elements, I would just write logic for two different cases: CMSDS + USWDS, and USWDS only

Reply:
Thanks Bernard - I believe I have this fixed up now